test/sqllogictest/chbench.slt

```diff
@@ -1353,46 +1295,48 @@ ORDER BY su_name
 | ArrangeBy (#4)
 
 6 =
-| Join %3 %4 %5 (= %4.#0 %5.#4)
+| Join %3 %4 %5 (= #1 #23)
 | | implementation = Differential %4 %5.(#4) %3.()
-| | demand for %3 = (#0)
-| | demand for %4 = (#0..#2)
-| | demand for %5 = (#6, #7)
-| Filter (datetots(#35) > 2010-05-23 12:00:00)
+| | demand = (#0..#3, #25, #26)
+| Filter (datetots(#25) > 2010-05-23 12:00:00)
 
 7 =
 | Get %2
 
 8 =
 | Get %6
-| Filter (#0 = ((#11 * #12) % 10000))
+| Filter (#0 = ((#1 * #2) % 10000))
 
 9 =
 | Get %6
-| Distinct group=(#11)
+| Distinct group=(#1)
 | ArrangeBy (#0)
 
 10 =
 | Get materialize.public.item (u24)
 | ArrangeBy (#0)
 
 11 =
-| Join %8 %9 %10 (= %8.#11 %9.#0 %10.#0)
-| | implementation = Differential %8 %9.(#0) %10.(#0)
-| | demand for %8 = (#0, #11..#13, #36)
-| | demand for %9 = ()
-| | demand for %10 = (#4)
-| Filter "^co.*$" ~(#44)
-| Reduce group=(#0, #11, #12, #13) sum(#36)
```

Comment:
This looks like a planning regression, where the joins could have been fused but were not.